packages/botonic-plugin-contentful/src/tools/validate-all-contents.ts

```diff
@@ -12,12 +12,12 @@ export class ContentsValidator {
    * - There's a bug on the CMS plugin
    */
   async validateAllTopContents(context: ContextWithLocale): Promise<void> {
-    // topContents will internally limit concurrency
-    const concurrency = context.concurrency && 1
-    await Parallel.each(
+    await asyncEach(
+      context,
       TOP_CONTENT_TYPES,
       ct => this.cms.topContents(ct, context),
-      concurrency
+      // topContents will internally manage concurrency
+      1
```

Comment:
If `topContents` will manage concurrency internally, will it be better if this parameter is ignored in this call in order to be undefined?

Reply:
If we specify undefined, it will request all content models simultaneously, which is what we don't want.
We want to get first all texts in parallel, then all carousels in parallel...That's why we request concurrency=1. Call me if it's not clear